### PR TITLE
Create explore_data_in_bigquery_with_workbench.ipynb

### PR DESCRIPTION
**Pull request by alokpattani**
Adding in notebook for exploratory data analysis as part of "Data to AI" effort. See this Colab for what this notebook looks like after it is run: https://colab.research.google.com/drive/1JeNeMtj2A_5P5vo9wxSkrwHQM5JQSoAu. Submitting it with outputs shown since a lot of this about interactive visualization, which can inspire folks to use/read the notebook beyond just the code.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/CODEOWNERS
Comment (ivanmkc) [inline notebooks/official/CODEOWNERS:33]: I think we accidentally deleted your line. Can you add your notebook back?

**Review by ivanmkc — APPROVED** [review]
LGTM after fixing CODEOWNERS.

**Comment by alokpattani** [comment]
<p>So there actually aren't any BQ resources to clean up since we use a public dataset and don't create any views, tables, models, etc. It's basically just different queries pulling data into notebooks and displaying as tables and charts in here, with outputs to Cloud Storage.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: notebooks/official/CODEOWNERS
Comment (alokpattani) [reply notebooks/official/CODEOWNERS]: Added it back at bottom.

**Comment by alokpattani** [comment]
<p>Acknowledged. Leaving it this time (for consistency - this naming has been used in other notebooks that may get published) but can consider for future functions.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>Made most/all of these changes.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>Went through and changed all "notebook" to lowercase except 1-2 where it goes with Jupyter (not Vertex) as something like a product title or display heading.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>Done.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>Done.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>Changed to "a set of dates"</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>The point I was making was more intricate (adding the "in" wasn't technically correct, but I get why it felt like it could be), so I ended up re-wording a couple parts of this paragraph altogether - I think it flows better now.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by alokpattani** [comment]
<p>Done.</p>

---

 <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/885/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**
